src/App.jsx

```diff
@@ -1,7 +1,53 @@
-import data from "./data.json";
+import { Album } from "./components/Album"
+import data from "./data.json"
 
-console.log(data);
+const selectAlbums = data.albums.items
 
-export const App = () => {
-  return <div>Find me in src/app.jsx!</div>;
+export const App = () =>{
+return(
+  <div>
+      <div className="albums">
+        {selectAlbums.map(album => (
+          <Album 
+            key={album.id}
+            artistName={album.artists[0].name}
```

Comment:
From requirements:

> You should display the following for each album:
> - the name of _each artist involved_ with a comma between

Reply:
All albums have just one artist, isn't it?

Reply:
Some have two (look at the second to last one)